src/main/webapp/css/main.css

```diff
@@ -16,18 +16,104 @@
 
 /* Prevent the page from becoming too wide. */
 body {
+	background: #5433FF;  /* fallback for old browsers */
+  background: -webkit-linear-gradient(to right, #A5FECB, #20BDFF, #5433FF);  /* Chrome 10-25, Safari 5.1-6 */
+  background: linear-gradient(to right, #A5FECB, #20BDFF, #5433FF); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+  color: #828c9b;
+	font-family: Product Sans;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
   max-width: 600px;
+  width: 50%;
```

Comment:
Does this make the width 50% of 600px or does it just set the width to 50% of the screen and then the max is 600px?

Reply:
This was a remnant from old code actually, it doesn't seem like its making a difference when I delete it